src/notation/qml/MuseScore/NotationScene/internal/PartialTiePopup.qml

```diff
@@ -0,0 +1,115 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2022 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Window 2.15
+
+import Muse.Ui 1.0
+import Muse.UiComponents 1.0
+import MuseScore.NotationScene 1.0
+
+StyledPopupView {
+    id: root
+    margins: 0
+
+    property NavigationSection notationViewNavigationSection: null
+    property int navigationOrderStart: 0
+    property int navigationOrderEnd: partialTieNavPanel.order
+
+    property QtObject model: partialTiePopupModel
```

Comment:
It might be good to make this and similar properties in other popups `readonly` and `alias`.

Reply:
`notationViewNavigationSection` and `navigationOrderStart` are set in ElementPopupLoader.qml, so can't be readonly.

Reply:
Sorry, I was unclear; I was referring to the `model` properties specifically here.